code/importers/cloc/setup.sh

```diff
@@ -1,4 +1,6 @@
 cd "$(dirname "$0")"
+command -v croc ||  echo "Install cloc and rerun"
+command -v croc ||  exit 125
```

Comment:
what does this line do?

Reply:
@SRS-WRKS

Reply:
Checks to see if program cloc is installed on the build machine.